Mastodon/Scene/Onboarding/Welcome/View/Pages/WelcomeContentPage.swift

```diff
@@ -49,6 +49,17 @@ enum WelcomeContentPage: CaseIterable {
         }
     }
     
+    var accessibilityLabel: String {
+        switch self {
+        case .whatIsMastodon:
+            return "\(L10n.Scene.Welcome.Education.WhatIsMastodon.title) Mastodon?"
```

Comment:
We could use an extra `Localizable.strings`-string here so that translation-folks can include "Mastodon" whereever they want, I'm thinking right-to-left-language

Reply:
Ah yeah totally! I'll update this 👍